src/components/Header.css

```diff
@@ -0,0 +1,82 @@
+header {
+  height: 42vh;
+  background-color: #3ad3c5;
+
+  /* opacity: 0.5; */
+}
+
+nav {
+  padding: 0% 2%;
+  background-color: #00455b;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+nav * {
+  color: white;
+}
+
+/* .header_img {
+    margin: auto;
+    height: 80%;
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+
+}
+
+.header_img img {
+    width: 60%;
+    height: 100%;
+}
+
+.header_img span {
+    justify-self: end;
+} */
+
+.fa-microphone {
```

Comment:
### Optional
- [ ] It would be better to remove all commented pieces of code

Reply:
thanks it's done